### PR TITLE
External db changes

### PR DESCRIPTION
**Pull request by mramendi**

### CODE REVIEW

File: doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc [hunk 1]
```diff
@@ -78,8 +74,8 @@ ifdef::PAM[]
 +
 .. *Dialect*: Enter the Hibernate dialect for the server, depending on the server type:
 +
-*** `org.hibernate.dialect.MySQL5InnoDBDialect` (used for MySQL and MariaDB)
-*** `org.hibernate.dialect.PostgreSQL82Dialect`
+*** `org.hibernate.dialect.MySQL5InnoDBDialect` (used for MySQL version 5.7 and for MariaDB)
+*** `org.hibernate.dialect.MySQL8Dialect` (used for MySQL version 8)
```
Comment (ruromero) [inline doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc:78]: Did you remove the PostgreSQL82 dialect by mistake?

Comment (spolti) [inline doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc:78]: IIRC Mariadb has it own dialect as well, 
for postgreeSQL use the dialect for version 10.

File: doc-content/enterprise-only/openshift/template-deploy-externaldb-proc.adoc [hunk 2]
```diff
@@ -47,7 +47,8 @@ complete the following additional configuration when configuring the template to
 ** *KIE Server External Database Host* (`KIE_SERVER_EXTERNALDB_SERVICE_HOST`) and  *KIE Server External Database Port* (`KIE_SERVER_EXTERNALDB_SERVICE_PORT`): The host name and port number of the external database server. You can set these parameters as an alternative to setting the `KIE_SERVER_EXTERNALDB_URL` parameter.
 ** *KIE Server External Database Dialect* (`KIE_SERVER_EXTERNALDB_DIALECT`): The Hibernate dialect for the server, depending on the server type:
 +
-*** `org.hibernate.dialect.MySQL5InnoDBDialect` (used for MySQL and MariaDB)
+*** `org.hibernate.dialect.MySQL5InnoDBDialect` (used for MySQL version 5.7 and for MariaDB)
+*** `org.hibernate.dialect.MySQL8Dialect` (used for MySQL version 8)
```
Comment (spolti) [inline doc-content/enterprise-only/openshift/template-deploy-externaldb-proc.adoc:51]: see comments above.

File: doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc (hunk 1 above)
Comment (mramendi) [reply doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc:78]: Yes, that's a mistake, I will readd

Comment (mramendi) [reply doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc:78]: I could not find a MariaDB dialect and a Postrges 10 dialect; as ths issue arose I'll await answers at https://issues.redhat.com/browse/BXMSDOC-6014 and put in the new dialect list in this PR

File: doc-content/enterprise-only/openshift/externaldb-build-proc.adoc [hunk 3]
```diff
@@ -6,18 +6,23 @@ If you want to use an external database server for a {KIE_SERVER} and the databa
 Complete the steps in this build procedure to provide drivers for any of the following database servers:
 
 * Microsoft SQL Server
-* MariaDB
 * IBM DB2
 * Oracle Database
 * Sybase
 
+You can also optionally use this procedure to build a new version of drivers for any of the following database servers:
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/externaldb-build-proc.adoc:13]: Minor suggestion - remove "also" and "any of", and maybe move "optionally".

Optionally, you can use this procedure to build a new version of drivers for the following database servers:

File: doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc [hunk 4]
```diff
@@ -60,7 +56,7 @@ ifdef::PAM[]
 ** `h2`: A built-in `h2` database engine that does not require a separate pod. Do not scale the {KIE_SERVER} pod if you use this setting.
 ** `external`: An external database server. 
 . If you selected any database except `external`, a Persistent Volume Claim will be created to store the database. Optionally, in the *Size* field, enter the size of the persistence volume to create.
-. If you selected the `external` database and you want to use any database server except PostgreSQL or MySQL, you must provide a {KIE_SERVER} extension image with the database server driver according to instructions in <<externaldb-build-proc>>. To configure the {KIE_SERVER} to use this extension image, make the following changes:
+. If you selected the `external` database, optionally configure the {KIE_SERVER} extension image. If you want to use any database server except PostgreSQL, MySQL, or MariaDB, you must provide a {KIE_SERVER} extension image with the database server driver according to instructions in <<externaldb-build-proc>>. To configure the {KIE_SERVER} to use this extension image, make the following changes:
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc:59]: Minor suggestion - Move optionally and rephrase:

Optionally, if you selected the `external` database, configure the {KIE_SERVER} extension image.

**Pull request merged**